css/index.css

```diff
@@ -48,99 +129,270 @@ table {
+	border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+	width: 48%;
+	padding: 0 1%;
+	padding-bottom: 20px;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+	margin-bottom: 20px;
+	border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+	padding: 0 2%;
+	margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	padding: 1%;
+}
+
+.middle-content .boxes .box1 {
+	background-color: teal;
+	padding: 5%;
+	margin: 2%;
+}
+.middle-content .boxes .box2 {
+	background-color: gold;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box3 {
+	background-color: cadetblue;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box4 {
+	background-color: coral;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box5 {
+	background-color: crimson;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box6 {
+	background-color: forestgreen;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box7 {
+	background-color: darkorchid;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box8 {
+	background-color: hotpink;
+	padding: 5%;
+	margin: 2%;
+}
+
+.middle-content .boxes .box9 {
+	background-color: indigo;
+	padding: 5%;
+	margin: 2%;
 }
 
-.middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+.middle-content .boxes .box10 {
+	background-color: dodgerblue;
+	padding: 5%;
+	margin: 2%;
 }
+/* .middle-content .boxes .box {
+	width: 12.5%;
+	height: 100px;
+	background: black;
+	margin: 20px 2.5%;
+	color: white;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+} */
 
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+	display: flex;
+	margin: 0 2% 20px;
+	justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+	padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+	padding-right: 0;
 }
 
+/* Footer CSS for both Home and About page Here */
 footer {
-    width: 100%;
-    background: black;
+	width: 100%;
+	background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+	width: 60%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px 2%;
+	font-size: 14px;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
-}
\ No newline at end of file
+	color: white;
+	text-decoration: none;
+}
+
+/* CSS for the About Page Starts Here */
+
```

Comment:
Some of these styles can be combined if you want to save some lines of CSS.  After I complete a project that uses CSS I always go back into the CSS files and see what I can combine that shares styles.